src/pybind/mgr/dashboard/frontend/src/app/shared/directives/copy2clipboard-button.directive.ts

```diff
@@ -33,21 +35,29 @@ export class Copy2ClipboardButtonDirective implements OnInit {
   onClick() {
     try {
       const browser = detect();
-      const text = this.getText();
+      const text = this.byId ? this.getText() : null;
```

Comment:
This will remove the need to check below:
```suggestion
      const text = this.byId ? this.getText() : this.cdCopy2ClipboardButton;
```

Reply:
Thank you. Made the changes @tspmelo